roles/kubernetes/node/tasks/main.yml

```diff
@@ -17,8 +17,11 @@
   tags:
     - kubelet
 
-- import_tasks: nginx-proxy.yml
-  when: is_kube_master == false and loadbalancer_apiserver_localhost
+- include: nginx-proxy.yml
```

Comment:
include is deprecated, you can use include_tasks or import_tasks difference is written here: http://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_includes.html

Reply:
ok . I will close it. I didn't switch branches